provider/provider_config/gcs_file_store_config_test.go

```diff
@@ -45,8 +47,8 @@ func TestGCSFileStoreDifferingFields(t *testing.T) {
 	if err != nil {
 		t.Errorf("failed to unmarshal GCP credentials: %v", err)
 	}
-	credentialsDict["client_email"] = "test@featureform.com"
-	gcpCredsBytesB, err := json.Marshal(credentialsDict)
+	credentialsDictB := createMapCopy(credentialsDict)
```

Comment:
Rename to something like "expectedCredentials"

Reply:
so it isn't the expected Credentials. Basically, this is the credential updated for gcs and we to make sure the original vs this one is different. I can update the names to `oldCredentialsDict` and `newCredentialsDict`.